packages/core/useContextMenu/component.ts

```diff
@@ -0,0 +1,29 @@
+import { defineComponent, h, reactive, ref } from 'vue-demi'
+import type { RenderableComponent, UseContextMenuOptions } from '@vueuse/core'
+import { useContextMenu } from '@vueuse/core'
+export interface useContextMenuProps extends UseContextMenuOptions, RenderableComponent {
+
+}
+
+export const UseContextMenu = defineComponent<useContextMenuProps>({
```

Comment:
Can useContextMenuProps be written directly as UseContextMenuOptions | RenderableComponent.